l1-contracts/test/foundry/l1/unit/concrete/state-transition/ChainTypeManager/ProtocolVersion.t.sol

```diff
@@ -6,9 +6,9 @@ import {Diamond} from "contracts/state-transition/libraries/Diamond.sol";
 import {ProtocolIdNotGreater} from "contracts/common/L1ContractErrors.sol";
 import {SemVer} from "contracts/common/libraries/SemVer.sol";
 import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
-
-import {console} from "forge-std/console.sol";
 import {Diamond} from "contracts/state-transition/libraries/Diamond.sol";
+import {ProtocolIdNotGreater} from "contracts/common/L1ContractErrors.sol";
+import {SemVer} from "contracts/common/libraries/SemVer.sol";
```

Comment:
duplicate imports (I have no idea why Solidity/our lints allow it)

Reply:
Removed.